tox.ini

```diff
@@ -18,6 +18,7 @@ description = run tests
 passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*
 deps =
     -rrequirements.txt
+    virtualenv!=20.0.2
```

Comment:
One more 2 at the end?

Reply:
ah, thanks!

Reply:
A similar type required this pinning too 😊 The first time on my side.